Update rootless.md (#19992)
docker/docs: content/engine/security/rootless.md

  script](https://get.docker.com/rootless), however, you must add an AppArmor
  profile for `rootlesskit` manually:

>   1. Create and install the currently logged-in user's AppArmor profile:
>      ``` console

Reviewer: formatting nit:

```suggestion
  1. Create and install the currently logged-in user's AppArmor profile:

     ```console
```

  script](https://get.docker.com/rootless), however, you must add an AppArmor
  profile for `rootlesskit` manually:

  1. Add the AppArmor profile to `/etc/apparmor.d/usr.local.bin.rootlesskit`:
  1. Create and install the currently logged-in user's AppArmor profile:

     ```console

>      $ cat <<EOF > /etc/apparmor.d/$(echo $HOME/bin/rootlesskit | sed -e s@^/@@ -e s@/@.@g)

Reviewer 2: Maybe just instruct to run the old commands as root

Author: Not all users have `sudo` access and the old command includes  `$HOME` which will always be `/root` when run as root. The old command appears to have never been and is not as versatile as the new.

> If the system-wide Docker daemon is already running, consider disabling it:
>```console
>$ sudo systemctl disable --now docker.service docker.socket
>$ sudo systemctl stop docker.service docker.socket
>```

> > Should you choose not to shut down the docker service and socket, you will need to use the `--force`

Reviewer: ```suggestion
> Should you choose not to shut down the `docker` service and socket, you will need to use the `--force`
```

> > parameter in the next section.  There are no known issues, but until you shutdown and disable you're
> > still running rooted Docker. 

Reviewer: ```suggestion
> parameter in the next section. There are no known issues, but until you shutdown and disable you're
> still running rootful Docker. 
```

> If the system-wide Docker daemon is already running, consider disabling it:
>```console
>$ sudo systemctl disable --now docker.service docker.socket

> >$ sudo systemctl stop docker.service docker.socket

Reviewer 2: Why doesn't `disable --now` work?

Author: I suppose it's just the docker socket is still present on the system.  `/var/run/docker.socket` needs to be forcefully removed in this step rather than simply `stop` on the service and socket.  This allows the user to run `dockerd-rootless-setuptool.sh install` without the `--force` parameter.

## Troubleshooting

### Errors installing the Docker daemon

> The `rootlesskit` cannot detect Systemd properly if you switch to your user via `sudo su`.  For users which cannot be logged-in, you must utilize the `machinectl` command which is a part of the `systemd-container` package. After installing `systemd-container` switch to "myuser" with the following command:

Reviewer: ```suggestion
`rootlesskit` cannot detect systemd properly if you switch to your user via `sudo su`. For users which cannot be logged-in, you must use the `machinectl` command which is part of the `systemd-container` package. After installing `systemd-container` switch to `myuser` with the following command:
```

## Troubleshooting

> ### Errors installing the Docker daemon

Author: Updated.
